Updates on woocommerce docs (#1125)
mercadopago/devsite-docs: guides/plugins/woocommerce/preferences.en.md

Activate the checkout you want to offer your customers according to your preferences and choose the payment methods with which they can make the purchase.

* Activate a checkout type.
  * Use Checkout Pro to support payments with cards, ----[mlb]---- Pix,------------ cash and money in the Mercado Pago account.

>   * Use the Custom Checkout to have control over other settings and accept payments with credit card, ----[mlb]---- Pix,------------ bank slip and in-person.

Reviewer: ```suggestion
  * Use the Custom Checkout to have control over other settings and accept payments with credit card, ----[mlb]---- Pix,------------ boleto and in-person.
```

>
> Nota
>

> > Check out the [Checkout Pro documentation](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/online-payments/checkout-pro/introduction/) to learn more about all its features and functionalities.

Reviewer: ```suggestion
> Check out the [Checkout Pro documentation](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/online-payments/checkout-pro/introduction) to learn more about all its features and functionalities.
```

| Discount coupon | Activate this option when you want to offer discounts to your customers. A field will appear on the form where they can enter their coupon. |
| Reduce inventory | Activate this option when you want to automate the inventory reduction for each purchase order that is approved after a payment with Mercado Pago. |

----[mlb]----

> #### Payments with PIX

Reviewer: ```suggestion
#### Payments with Pix
```

| Configuration | Description |
| --- | --- |

> | PIX Expiration | Set the expiration time for the code sent to the customer after the order is placed. This is the period in which the customer will have to pay for the purchase. |

Reviewer: ```suggestion
| Pix Expiration | Set the expiration time for the code sent to the customer after the order is placed. This is the period in which the customer will have to pay for the purchase. |
```

> | Pix Expiration | Set the expiration time for the code sent to the customer after the order is placed. This is the period in which the customer will have to pay for the purchase. |

Reviewer: ```suggestion
| Pix expiration | Set the expiration time for the code sent to the customer after the order is placed. This is the period in which the customer will have to pay for the purchase. |
```
